kegtab/src/main/res/layout/tap_detail_loaded.xml

```diff
@@ -55,10 +55,25 @@
         </org.kegbot.app.view.BadgeView>
     </LinearLayout>
 
+    <TextView
+        android:id="@+id/tapNotes"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/tapStatsBadges"
+        android:layout_centerHorizontal="true"
+        android:layout_margin="16dp"
+        android:layout_marginTop="2dip"
+        android:padding="16dp"
```

Comment:
and padding.

Reply:
Looked ok on my tablet...but looks better with it eliminated/reduced! Fixed!